rothschild/src/main.rs

```diff
@@ -165,7 +225,7 @@ async fn main() {
         }
         if !has_funds || now - last_refresh > 60_000 {
             info!("Refetching UTXO set");
-            tokio::time::sleep(Duration::from_millis(100)).await; // We don't want this operation to be too frequent since its heavy on the node, so we wait some time before executing it.
+            tokio::time::sleep(Duration::from_fmillis(100)).await; // We don't want this operation to be too frequent since its heavy on the node, so we wait some time before executing it.
```

Comment:
Likely a typo and should just be `from_millis`. I get this when I build:

```
error[E0599]: no function or associated item named `from_fmillis` found for struct `Duration` in the current scope
   --> rothschild/src/main.rs:228:42
    |
228 | ...   tokio::time::sleep(Duration::from_fmillis(100)).await; // We don't want this operation to be too frequent since its heavy on the node, so we wa...
    |                                    ^^^^^^^^^^^^
    |                                    |
    |                                    function or associated item not found in `Duration`
    |                                    help: there is an associated function with a similar name: `from_millis`
```